nss-day-cohort-46/rare-server-tragic-trolls
# Nc approve posts

# Description
Allows a post.approved to be changed from true to false.
## Type of change


# Testing Instructions

Ensure you have a post that is unapproved. If you do not have one:

```
INSERT INTO Posts ('id', 'user_id', 'category_id', 'title', 'publication_date', 'image_url', 'content', 'approved') VALUES (Null, 2, 1, 'Title#6', '2026-04-02T23:57:07.374Z', '', 'APPROVE ME', 0);
```

Do a PATCH request in Postman:
```
http://localhost:8088/approve/THE-ID-OF-YOUR-POST-GOES-HERE
```

Do a GET request in Postman:
```
http://localhost:8088/posts/THE-ID-OF-YOUR-POST-GOES-HERE
```

See if the approval returns a true boolean

# Checklist:
